### DISCUSSION TITLE
Windows Store - 10.1.1.11 Inaccurate Representation - Icon

### DISCUSSION THREAD

**Discussion by Andrew-web-coder** [alex8088/electron-vite#206] [Q&A]
Hello, 

I need help getting published to Windows Store, I have already tried multiple times, but it it always gets rejected due to this issue:

```
10.1.1.11 Inaccurate Representation - Icon 

There is an issue with the functionality of the product. An additional installed component of the product visible in the Start Menu does not work as expected.
```

So, I have successfully build my setup.exe (using `build:win` command), it works perfectly fine to me - all icons look fine, all shortcuts work fine! My app does not install anything additionally and does not create additional icons, just the one needed for the app.

Then I use `MSIX Packaging Tool` to create `msix` package and then I submit to Microsoft Partner Center, but without any success.

Could someone help me, please?

btw, I also extracted the package, inspected the content and did not found any issue, it also contains tons of icons. I have created and placed icon.icns, icon.ico, icon.png inside `build` directory, I think it should be enough, am I missing something? Sadly, I am not able to instal the package, because it is not signed with the certificate.

**Comment by alex8088** [comment]
https://www.electron.build/configuration/appx.html#how-to-publish-your-electron-app-to-the-windows-app-store

**Reply by Andrew-web-coder** [reply]
@alex8088 The link provided there does not exist. "build and test your appx. It’s dead simple" <- I have no idea how to build and test appx.

**Comment by Andrew-web-coder** [comment]
The answer is - it is a bug of "MSIX Packaging Tool" and use "appx" format.
